src/libstd/time.rs

```diff
@@ -59,6 +59,21 @@ pub use core::time::Duration;
 ///    println!("{}", now.elapsed().as_secs());
 /// }
 /// ```
+///
+/// # Underlying System calls
+/// Currently, the following system calls are being used to get the current time using `now()`:
+///
+/// |  Platform |               System call                        |
+/// |:---------:|:------------------------------------------------:|
+/// | Cloud ABI | [clock_time_get (Monotonic Clock)](https://github.com/NuxiNL/cloudabi/blob/master/cloudabi.txt) |
+/// |    SGX    | Not implemented                                  |
+/// |    UNIX   | [mach_absolute_time](https://developer.apple.com/library/archive/documentation/Darwin/Conceptual/KernelProgramming/services/services.html)                           |
```

Comment:
This line should be Darwin, UNIX probably uses `clock_gettime` as in the next line (didn't check).

Reply:
Indeed, Unix targets other than Mac and iOS use `clock_gettime` (with `CLOCK_MONOTONIC`).

Reply:
The code for this was in the "sys/unix/time.rs" however. Should I still change it to Darwin? `mach_absolute_time` seems to be a Mac specific system call.

Reply:
Ahhh never mind, I see some macos architecture tags now. Thanks!